src/app/containers/RadioPageMain/index.stories.jsx

```diff
@@ -2,10 +2,10 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { inputProvider } from '@bbc/psammead-storybook-helpers';
 import { withKnobs } from '@storybook/addon-knobs';
+import WithTimeMachine from '../../../testHelpers/withTimeMachine';
```

Comment:
Does the alias work here?
```suggestion
import WithTimeMachine from '#testHelpers/withTimeMachine';
```

Reply:
Yes.